physics.py

```diff
@@ -16,22 +16,24 @@
 
 
 class SimpleTalonFXMotorSim:
-    def __init__(self, motor: phoenix5.TalonFX, kV: float, rev_per_unit: float) -> None:
-        self.sim_collection = motor.getSimCollection()
+    def __init__(
+        self, motor: phoenix6.hardware.TalonFX, kV: float, rev_per_unit: float
+    ) -> None:
+        self.sim_collection = phoenix6.sim.TalonFXSimState(motor)
         self.kV = kV  # volt seconds per unit
         self.rev_per_unit = rev_per_unit
 
     def update(self, dt: float) -> None:
-        voltage = self.sim_collection.getMotorOutputLeadVoltage()
+        voltage = self.sim_collection.motor_voltage()
         velocity = voltage / self.kV  # units per second
         velocity_cps = velocity * self.rev_per_unit * FALCON_CPR
-        self.sim_collection.setIntegratedSensorVelocity(int(velocity_cps / 10))
-        self.sim_collection.addIntegratedSensorPosition(int(velocity_cps * dt))
+        self.sim_collection.set_rotor_velocity(int(velocity_cps))
+        self.sim_collection.add_rotor_position(int(velocity_cps * dt))
 
 
 class SimpleTalonSRXMotorSim:
     def __init__(
-        self, motor: phoenix5.TalonSRX, kV: float, rev_per_unit: float
+        self, motor: phoenix6.hardware.TalonSRX, kV: float, rev_per_unit: float
```

Comment:
I thought the Talon SRX isn't a Phoenix 6 device?

Reply:
Yes, TalonSRX is pheonix5 only. I just changed all occurrences of pheonix5 to pheonix6. Haven't got time to change the SRX yet.